### PR TITLE
Catagory model Updated

### PR DESCRIPTION
**Pull request by spartha1995**
File Changed :TrappistDbContext.cs

### CODE REVIEW

File: Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using Microsoft.EntityFrameworkCore;
 using Promact.Trappist.Web.Models;
```
Comment (kinjalgor) [inline Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs]: test

Comment (spartha1995) [inline Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs]: If i remove EntityFrameworkCore then DbContextOptions,OnModelCreating,DbSet throwing error.
and if i remove web.Model then ApllicationUser is Throwing Error Messege

Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs]: I said unused using. Please read about unused using statements.

Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs]: why you have removed namespace? i asked you to update it as per feature, not to remove it.

Comment (Krunal-Promact) [inline Trappist/src/Promact.Trappist.DomainModel/DbContext/TrappistDbContext.cs]: you have again added wrong namespace. please correct the namespace

**Pull request closed without merging**